src/_P092_DLbus.ino

```diff
@@ -423,34 +354,38 @@ boolean Plugin_092(uint8_t function, struct EventStruct *event, String& string)
 
     case PLUGIN_INIT:
     {
+      String log = F("PLUGIN_092_INIT Task:");
+      log += event->TaskIndex;
+      addLog(LOG_LEVEL_INFO, log);
       if (P092_init) {
-        addLog(LOG_LEVEL_ERROR, F("P092_init -> Already done!"));
+        addLog(LOG_LEVEL_INFO, F("INIT -> Already done!"));
       }
       else {
-        addLog(LOG_LEVEL_INFO, F("P092_init ..."));
+        if (P092_data == nullptr) {
```

Comment:
i'm not quite sure whether re-initialization is done in case GPIO pin is changed because of this query.

P092_init is set to true in PLUGIN_WEBFORM_SAVE but i think P092_data is still != nullptr and therefore the initialization function is not called.